scripts/analysis/index.ts

```diff
@@ -0,0 +1,143 @@
+import { repos } from './config.json';
```

Comment:
i get `scripts/analysis/index.ts:1:23 - error TS2307: Cannot find module './config.json' or its corresponding type declarations.` when i try to run `yarn get-usage`?

Reply:
Ah yes. There's a `config.json` file that you need which I'll send over. I put it in the `.gitignore` because I wasn't sure it should live in the repo. Longer term I was thinking about putting it in s3 and then loading it from there using Janus creds?

Reply:
I can't think of a reason why it shouldn't live in the repo, at least temporarily? There's no secrets from what I can tell. Just a lovely bit of hard-coding 😊

Reply:
"HARD CODE AND COMMIT"

Reply:
Should be there now 📄